### PR TITLE
Send perf tests results to ci database

### PR DESCRIPTION
**Pull request by vdimir**
### Changelog category (leave one):

- Not for changelog (changelog entry is not required)

### Changelog entry (a user-readable short description of the changes that goes to CHANGELOG.md):
...

> Information about CI checks: https://clickhouse.com/docs/en/development/continuous-integration/

### CODE REVIEW

File: tests/ci/performance_comparison_check.py [hunk 1]
```diff
@@ -133,6 +133,9 @@ def __exit__(self, exc_type, exc_val, exc_tb):
     if not os.path.exists(result_path):
         os.makedirs(result_path)
 
+    docker_env += (
+        f" -e CHPC_DATABASE_URL -e CHPC_DATABASE_USER -e CHPC_DATABASE_PASSWORD"
```
Comment (alexey-milovidov) [inline tests/ci/performance_comparison_check.py:137]: Let's rename to `CLICKHOUSE_PERFORMANCE_COMPARISON_DATABASE_URL` and similar.

**Comment by vdimir** [comment]
Some result uploaded, but not all 

Looks like we upload only failed or unstable queries
https://github.com/ClickHouse/ClickHouse/blob/9d67fff640ca629b36faa13ccf6eed6825cbb5f6/docker/test/performance-comparison/compare.sh#L1237

https://play.clickhouse.com/play?user=play#U0VMRUNUICogRlJPTSBjaGVja3MgV0hFUkUgY2hlY2tfbmFtZSA9ICdQZXJmb3JtYW5jZScKT1JERVIgQlkgY2hlY2tfc3RhcnRfdGltZSBERVND

Also I added more verbose check_name with `RUN_BY_HASH` and `CHECK_NAME`

File: docker/test/performance-comparison/compare.sh [hunk 2]
```diff
@@ -1285,7 +1306,6 @@ create table ci_checks engine File(TSVWithNamesAndTypes, 'ci-checks.tsv')
                     test text, query_index int, query_display_name text')
             settings date_time_input_format='best_effort'
             format TSV
-            settings date_time_input_format='best_effort'
```
Comment (Felixoid) [inline docker/test/performance-comparison/compare.sh:1288]: Just curious, why it's removed?

File: docker/test/performance-comparison/compare.sh [hunk 3]
```diff
@@ -1312,6 +1332,8 @@ $REF_SHA	$SHA_TO_TEST	$(numactl --show | sed -n 's/^cpubind:[[:space:]]\+/numact
 $REF_SHA	$SHA_TO_TEST	$(numactl --hardware | sed -n 's/^available:[[:space:]]\+/numactl-available	/p')
 EOF
 
+    head ci-checks.tsv
```
Comment (Felixoid) [inline docker/test/performance-comparison/compare.sh:1335]: Is it debugging thing?

**Review by Felixoid — COMMENTED** [review]
Looks good to me besides some points

File: docker/test/performance-comparison/compare.sh (hunk 3 above)
Comment (vdimir) [reply docker/test/performance-comparison/compare.sh:1335]: Yes, we can remove it

File: docker/test/performance-comparison/compare.sh [hunk 4]
```diff
@@ -1312,6 +1332,8 @@ $REF_SHA	$SHA_TO_TEST	$(numactl --show | sed -n 's/^cpubind:[[:space:]]\+/numact
...
+
```
Comment (vdimir) [inline docker/test/performance-comparison/compare.sh:1336]:
```suggestion
```

File: docker/test/performance-comparison/compare.sh (hunk 2 above)
Comment (Felixoid) [reply docker/test/performance-comparison/compare.sh:1288]: Maybe remove another too then? =)

File: docker/test/performance-comparison/compare.sh [hunk 5]
```diff
@@ -1285,7 +1306,6 @@ create table ci_checks engine File(TSVWithNamesAndTypes, 'ci-checks.tsv')
                     test text, query_index int, query_display_name text')
             settings date_time_input_format='best_effort'
```
Comment (vdimir) [inline docker/test/performance-comparison/compare.sh:1307]:
```suggestion
```

File: docker/test/performance-comparison/compare.sh (hunk 2 above)
Comment (vdimir) [reply docker/test/performance-comparison/compare.sh:1288]: Done

**Pull request merged**